bin/delete-coauthor.js

```diff
@@ -0,0 +1,24 @@
+#! /usr/bin/env node
+const minimist = require('minimist');
+const { runDeleteCoauthorHelp } = require('../src/helpers');
+const { deleteCoauthor } = require('../src/manage-authors/delete-coauthor');
+
+const argv = minimist(process.argv.slice(2), {
+  alias: {
+    h: 'help',
+  },
+});
+
+async function execute(argv) {
+  if (argv.help) {
+    runDeleteCoauthorHelp();
+    process.exit(0);
+  }
+
+  const args = argv._;
+
+  await deleteCoauthor(args[0]);
```

Comment:
Passing no initials `git delete-coauthor` returns `undefined: no such initials in .git-coauthors`

I think it should show a message saying please provide initials for co-author to delete or something along those lines.